src/main/webapp/map.html

```diff
@@ -5,11 +5,20 @@
     <link rel = "stylesheet" href="/css/map.css">
     <script src="/js/map.js"></script>
     <title>Map Page</title>
-    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCEr8IFDb0oorKv9HmcL-XDPu62EAa8kRY"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    
   </head>
   <body onload="createMap();">
     <h1>Google Map API</h1>
     <div id="map"></div>
     <p>This page shows a Google Map using a fetch method to get data!</p>
+    <div id="locationField">
+      <input id="autocomplete"
+         placeholder="Enter your address"
+         onFocus="geolocate()"
+         type="text"/>
+    </div>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCEr8IFDb0oorKv9HmcL-XDPu62EAa8kRY&libraries=places&callback=initAutocomplete"
```

Comment:
<script> tags should typically be grouped together in the <head> tag for easier readability. Right now the <script>s are kind of spread out